partner_contact_nutrition_exclusion/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    nutrition_exclusion_ids = fields.Many2many(comodel_name='product.product',
```

Comment:
Not strictly wrong, but much better to start all arguments on next line, so you can keep indents to 4 spaces, instead of have everything in a narrow space near the end of the line.